docs/content/tools/xdebug.md

```diff
@@ -138,31 +138,43 @@ You can run your scripts in console and debug them in the same way as browser re
           "cwd": "${fileDirname}",
           "port": 9000,
           "pathMappings": {
-            "/var/www/": "${workspaceRoot}"
+            "/var/www/": "${workspaceFolder}"
           }
         }
       ]
     }
     ```
-    
+
 4. Set a breakpoint
 5. Select the **Listen for XDebug** configuration from the dropdown and click **Start Debugging**
 
 You can debug both web requests and cli scripts using this configuration.
 
+### CLI Scripts: Drush {#vscode-drush}
+
+To debug Drush commands using Xdebug and VSCode add the following to your path mappings
+
+**Drush 8.x**
+
+- Add `"/usr/local/bin/drush": "${workspaceRoot}/bin/drush"` to `pathMappings` in your `launch.json` file.
+
+**Drush 9.x**
+
+- Add `"/usr/local/bin/drush": "${workspaceRoot}/vendor/bin/drush"` to `pathMappings` in your `launch.json` file.
```

Comment:
Is it `workspaceRoot` or `workspaceFolder`?

Reply:
Fudge... missed the main reason I started this.  Good catch!